apps/website/src/server/db/giveaways.ts

```diff
@@ -0,0 +1,130 @@
+import { z } from "zod";
+import type { GiveawayEntry } from "@prisma/client";
+import { getCountryName, isValidCountryCode } from "@/utils/countries";
+import { prisma } from "@/server/db/client";
+import { decryptRecord, encryptRecord } from "@/server/db/encryption";
+import type { GiveawayEntryWithAddress } from "@/pages/giveaways/[giveawayId]";
+
+const entryEncryptFields = ["givenName", "familyName"];
+
+const mailingAddressEncryptFields = [
+  "addressLine1",
+  "addressLine2",
+  "postalCode",
+  "city",
+  "state",
+];
```

Comment:
add country?!

Reply:
👀 Yeah I would assume so, anything identifiable about the user